drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5126,7 +5126,7 @@ void RasterizerStorageGLES2::_render_target_allocate(RenderTarget *rt) {
 	}
 
 	// Allocate mipmap chains for post_process effects
-	if (!rt->flags[RasterizerStorage::RENDER_TARGET_NO_3D] && rt->width >= 2 && rt->height >= 2) {
+	if (!rt->flags[RasterizerStorage::RENDER_TARGET_NO_3D] && !rt->flags[RasterizerStorage::RENDER_TARGET_NO_MIPMAPS] && rt->width >= 2 && rt->height >= 2) {
```

Comment:
These are the mipmaps for post processing effects like glow. I don't think they should be disabled when "no mipmaps" is selected.

Reply:
Yep that's what I intended.  Post process effects should still run, just not with mipmap sampler.  

See https://github.com/godotengine/godot/pull/53275/files#diff-e61a2716371ede6bb16219d6b8f676a851fdeb08b948798cf11a4a0041eb6808R681-R688

Gles2 I might have to take another look at....  As it looks a little different on second look.  But that's what currently happening on gles3.

> In my opinion the best approach would be to just store a single flag called ``generate_mipmaps" or something similar that can be toggled on and off.

RENDER_TARGET_NO_MIPMAPS is basically this flag...  Or you would want another separate flag per viewport?  Either way, there would need to be a flag on RenderTarget, but it doesn't matter to me how it's exposed externally in the viewport.  It looked like viewport usage_mode is design to moderate the RenderTarget flag permutations here:

https://github.com/godotengine/godot/pull/53275/files#diff-e61a2716371ede6bb16219d6b8f676a851fdeb08b948798cf11a4a0041eb6808R657-R698

Reply:
Ya I can confirm glow is broken without mipmaps.  The main issue is, when keeping locality to just the SCREEN_TEXTURE issue, you still have to end up setting up the screen's RenderTarget correctly, which ur right affects glow.  hmmmm